express/scripts/scripts.js

```diff
@@ -331,6 +331,21 @@ export function readBlockConfig($block) {
   return config;
 }
 
+/**
+ * this is an extensible stub to take on audience mappings
+ * @param {string} audience
+ * @return {boolean} is member of this audience
+ */
+function checkExperimentAudience(audience) {
+  if (audience === 'mobile') {
+    return window.innerWidth < 600;
+  }
+  if (audience === 'desktop') {
+    return window.innerWidth > 600;
+  }
+  return true;
```

Comment:
This changes the current implementation from user agent based to window width based, which will impact existing content intended for mobile `devices` (e.g. tablets) which should see the same content as mobiles, but have overlapping window widths with desktops...

Reply:
I did not introduce this method (just moved it from the bottom of the file to the top) but I still broke the logic by re-using this method in the wrong place. Fixing now.